libs/entitlement/src/main/java/org/elasticsearch/entitlement/runtime/policy/FileAccessTree.java

```diff
@@ -53,8 +54,12 @@ boolean canWrite(Path path) {
         return checkPath(normalize(path), writePaths);
     }
 
+    private static String normalizedPath(FileEntitlement fileEntitlement) {
+        return normalize(Paths.get(fileEntitlement.path()));
+    }
+
     private static String normalize(Path path) {
-        return path.toAbsolutePath().normalize().toString();
+        return path.toAbsolutePath().normalize().toString().replace('\\', '/');
```

Comment:
Is this correct with the root path on Windows, eg `C:\` into `C:/`?

Reply:
Oh and error messages / logs I guess.

Reply:
Error messages and logs are kind of important, it needs to be something a user will understand, right?

Reply:
Yep, that's why I mentioned it.

`toAbsolutePath()` converts slashes to backslashes. I suppose we could rely on each system's `toAbsolutePath()` to use `File.separator` and use that.

Reply:
Why isn't `toAbsolutePath()` good enough for normalization, since we use it on both sides?

Reply:
It is. That's what I'm suggesting.